metrics/xrootd_metrics.go

```diff
@@ -308,6 +316,11 @@ var (
 		Help: "Number of bytes the data requested is in the cache or not",
 	}, []string{"path", "type"}) // type: hit/miss/bypass
 
+	CacheIO = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
I don't think the `Gauge` type is correct for all the data here.
- active I/O is the current number of I/O operations ongoing.
- total I/O is an operation counter: the total number of I/O operations seen by the throttle plugin.  This should be monotonically increasing.
- I/O wait time is the total time that the server was waiting on operations to complete.  This should be monotonically increasing.

So, a gauge type is appropriate for the first two but not the latter two.